src/app/shared/pipes/relativeTime.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DAYS, HOURS, MINUTES, MONTHS } from '../helpers/duration';
+import { LocaleService } from '../../core/services/localeService';
+
+const formatTime = (locale?: string) => (value: string | Date): string => {
+  const valueDate = new Date(value);
+  if (isNaN(valueDate.getTime())) {
+    throw new Error(`Unexpected: Invalid date value: '${ value.toString() }'`);
+  }
+
+  const currentDate = new Date();
+  const currentLocale = locale || 'en';
+
+  const diffInMS = valueDate.getTime() - currentDate.getTime();
+  const formatter = new Intl.RelativeTimeFormat(currentLocale, { numeric: 'auto' });
+  const absDiffInMS = Math.abs(diffInMS);
+
+  if (absDiffInMS < MINUTES) {
+    return $localize`Just now`;
+  }
+
+  if (absDiffInMS > MINUTES && absDiffInMS < HOURS) {
+    return formatter.format(Math.trunc(diffInMS / MINUTES), 'minutes');
+  }
+
+  if (absDiffInMS > HOURS && absDiffInMS < DAYS) {
+    return formatter.format(Math.trunc(diffInMS / HOURS), 'hours');
```

Comment:
is it internationalized?

Reply:
The `Intl.RelativeTimeFormat` object enables language-sensitive relative time formatting - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat

I've seen fr lang but I can't check if it's correct. But I assume as it part of js - must support the cases you described (plural)